lib/classes/TradeOffer.js

```diff
@@ -248,19 +255,22 @@ TradeOffer.prototype.removeTheirItems = function(items) {
 
 function addItem(details, offer, list) {
 	if (offer.id) {
-		throw new Error("Cannot add items to an already-sent offer");
+		throw new Error('Cannot add items to an already-sent offer');
 	}
 
-	if (typeof details.appid === 'undefined' || typeof details.contextid === 'undefined' || (typeof details.assetid === 'undefined' && typeof details.id === 'undefined')) {
-		throw new Error("Missing appid, contextid, or assetid parameter");
+	if (typeof details.appid === 'undefined'
+		|| typeof details.contextid === 'undefined'
+		|| (typeof details.assetid === 'undefined'
+			&& typeof details.id === 'undefined')) {
```

Comment:
For this case and all simular cases: there's a rule in ESLint that enforces the max length of the line. I agree with it, cause for me having a 300 symbols line isn't good. But this also looks weird, I agree with you.

@DoctorMcKay do you know any other ways that it can be solved? Maybe move it into a separate helper function, like I did with `TradeOffer.needsConfirmation`? Not sure about it.